spec/spec_helper_without_loading_attr_extras.rb

```diff
@@ -2,3 +2,23 @@
 require "minitest/pride"
 
 $: << File.dirname(__FILE__) + "/../lib"
+
+module AttrExtras
+  module Assertions
+    def refute_warnings_emitted
+      original_stderr = $stderr
+      $stderr = StringIO.new
+      yield
+      assert $stderr.string.empty?, -> do
+        warnings = $stderr.string.strip.split("\n").map { |line| "  #{line}" }.join("\n")
+        "Expected no warnings to be emitted, but these were:\n\n#{warnings}"
+      end
+    ensure
+      $stderr = original_stderr
+    end
+  end
+end
+
+Minitest::Test.class_eval do
+  include AttrExtras::Assertions
```

Comment:
I haven't dabbled in Minitest in a while so I hope it's okay that I'm mixing this module into Minitest::Test! If there's a better way to do this (or you want me to change this entirely) let me know.